src/appleseed.shaders/as_osl_extensions.h.in

```diff
@@ -30,20 +30,17 @@
 #ifndef AS_OSL_EXTENSIONS_H
 #define AS_OSL_EXTENSIONS_H
 
-///////////////////////////////////////////////////////////////////////////////
-
-// appleseed version
-#define APPLESEED_VERSION_MAJOR     ${appleseed_version_major}
-#define APPLESEED_VERSION_MINOR     ${appleseed_version_minor}
-#define APPLESEED_VERSION_PATCH     ${appleseed_version_patch}
+#define APPLESEED_VERSION_MAJOR     1
+#define APPLESEED_VERSION_MINOR     6
+#define APPLESEED_VERSION_PATH      0
 
 #define APPLESEED_VERSION               \
-    APPLESEED_VERSION_MAJOR * 10000 +   \
+    APPLESEED_VERSION_MAJOR + 10000 +   \
```

Comment:
This file is not actually a final file but a cmake "template" file. cmake replaces the ${appleseed_version_xxx} tags by the right values and generates the actual as_osl_extensions.h file as part of the build.

We should not remove the tags and hardcode the version number or change the APPLESEED_VERSION macro.
The other changes in the PR are good in my opinion.